src/makers/hotspot/rak.ts

```diff
@@ -0,0 +1,41 @@
+import RAK from '@assets/images/rak.svg'
+import { MakerHotspot } from './hotspotMakerTypes'
+import RAK_ANTENNAS from '../antenna/rak'
+
+const RAK_V2 = {
+  name: 'RAK v2 Hotspot',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Helium to identify issues with your Hotspot in a secure way.</white></b>\n\nHelium will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
+      power: ['Placeholder', 'Placeholder'],
+      bluetooth: [
+        'Press the black button on your Hotspot. Its light should turn blue.',
+        'Placeholder',
+      ],
+    },
+    ko: {},
+    ja: {},
+    zh: {},
+  },
+  icon: RAK,
+  antenna: { us: RAK_ANTENNAS.RAK_US, default: RAK_ANTENNAS.RAK_EU },
```

Comment:
Should default be EU?

Reply:
I think so. The code checks `const isUS = country === 'US'` and if so it picks the `us` configuration.

Reply:
Hmm maybe we call it EU and the code just defaults to EU? I did it that way because I couldn't find a good way to determine EU since there are many countries. So we just check for US and then if its not use EU since they all had only 2. I guess if there were 3 it would be different.

Reply:
Actually I guess default makes sense haha, I guess it only really becomes an issue if there are more than 2 types which I'm not sure will happen